### ISSUE TITLE
Boom! 💥 open ./internal/report/assets/template.html: no such file or directory

### ISSUE THREAD

**Issue by tomislater** [derailed/popeye#85]
<img src="https://raw.githubusercontent.com/derailed/popeye/master/assets/popeye_bug.png" align="right" width="100" height="auto"/>

<br/>
<br/>
<br/>

**Describe the bug**
I cannot create html report.

**To Reproduce**
```
➜ POPEYE_REPORT_DIR=$(pwd) popeye -o html --save --namespace mynamespace --sections pod
/Users/myuser/sanitizer_cluster.private-cluster.com_1583144387909932000.html
 ___     ___ _____   _____                       D          .-'-.
| _ \___| _ \ __\ \ / / __|                       O     __| K    `\
|  _/ _ \  _/ _| \ V /| _|                         H   `-,-`--._   `\
|_| \___/_| |___| |_| |___|                       []  .->'  X     `|-'
  Biffs`em and Buffs`em!                            `=/ (__/_       /
                                                      \_,    `    _)
                                                         `----;  |


Boom! 💥 open ./internal/report/assets/template.html: no such file or directory
```

I have found this `template.html` here: https://github.com/derailed/popeye/blob/a96ac4a7ecf6aaea7ff705e6e51a020aec2110e4/internal/report/builder.go#L161

**Expected behavior**
HTML report should be saved.

**Versions (please complete the following information):**
 - OS: OSX and Linux
 - Popeye 0.7.0
 - K8s 1.14+

**Comment by derailed** [comment]
@tomislater Thank you for this report! My bad! I'll fix in the next drop.

**Comment by derailed** [comment]
@tomislater Fixed v0.7.1! Sorry ;(
